go.work.sum

```diff
@@ -0,0 +1,18 @@
+github.com/deckarep/golang-set v1.8.0 h1:sk9/l/KqpunDwP7pSjUg0keiOOLEnOBHzykLrsPppp4=
```

Comment:
Should probably remove this file

Reply:
Yeah it seems old. It's holding the old version of the protobufs.. If not you can try to update it but just remove IMHO.

Reply:
Done